test/index.js

```diff
@@ -445,7 +805,27 @@ describe('def', function() {
   });
 
   it('supports custom types', function() {
-    var env = $.env.concat([Integer, $Pair]);
+
+    //  anonJust :: a -> Just a
```

Comment:
Should the type be `a -> Maybe a` rather than `a -> Just a`? Also, should we name the function `AnonJust` rather than `anonJust`, to follow the convention of capitalizing the names of data constructors? Similarly, should we name the type `AnonMaybe`?